components/dashboard/src/admin/ProjectDetail.tsx

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License-AGPL.txt in the project root for license information.
+ */
+
+import moment from "moment";
+import { Link } from "react-router-dom";
+import { Project } from "@gitpod/gitpod-protocol";
+import Prebuilds from "../projects/Prebuilds"
+import Property from "./Property";
+
+export default function ProjectDetail(props: { project: Project, owner: string | undefined }) {
+    return <>
+        <div className="flex">
+            <div className="flex-1">
+                <div className="flex"><h3>{props.project.name}</h3><span className="my-auto ml-3"></span></div>
+                <p>{props.project.cloneUrl}</p>
+            </div>
+        </div>
+        <div className="flex mt-6">
+            <div className="flex flex-col w-full">
+                <div className="flex w-full mt-6">
+                    <Property name="Created">{moment(props.project.creationTime).format('MMM D, YYYY')}</Property>
+                    <Property name="Repository"><a className="text-blue-400 dark:text-blue-600 hover:text-blue-600 dark:hover:text-blue-400 truncate" href={props.project.cloneUrl}>{props.project.name}</a></Property>
+                    {props.project.userId ?
+                        <Property name="Owner">
+                            <Link className="text-blue-400 dark:text-blue-600 hover:text-blue-600 dark:hover:text-blue-400 truncate" to={"/admin/users/" + props.project.userId}>{props.owner} (User)</Link>
+                        </Property>
+                        :
+                        <Property name="Owner">{`${props.owner} (Team)`}</Property>}
+                </div>
+                <div className="flex w-full mt-6">
+                    <Property name="Incremental Prebuilds">{props.project.settings?.useIncrementalPrebuilds ? "Yes" : "No"}</Property>
+                    <Property name="Marked Deleted" >{props.project.markedDeleted ? "Yes" : "No"}</Property>
```

Comment:
**issue**: This is minor but the column alignment seems slightly off for this element. We probably don't need the `ml-3` class here or the first child elements for each row, at least until we start working on more complex responsive layouts. What do you think?

| BEFORE | AFTER |
|-|-|
| <img width="968" alt="before" src="https://user-images.githubusercontent.com/120486/151860199-ae3b32fa-0642-45ae-a325-33bb430fdbb2.png"> | <img width="968" alt="after" src="https://user-images.githubusercontent.com/120486/151860204-0c762a8a-5913-44d1-b73a-7b6cca258c20.png"> |

Reply:
I just did it in a hacky way and `markedDeleted` is still off 😅 Feel free to share your CSS magic with me �
<img width="745" alt="Screenshot 2022-01-31 at 22 48 55" src="https://user-images.githubusercontent.com/8015191/151878720-6666b434-195e-4765-8be8-e10e10bf7ee9.png">
�

Reply:
Sorry I think your comment came at the same time as the merging! Will do this as a drive-by in the next admin ticket 🙏🏽

Reply:
Thanks, @laushinka! Took a bias-for-action and opened https://github.com/gitpod-io/gitpod/pull/7960 for this minor fix.